.github/workflows/production.yaml

```diff
@@ -30,8 +30,15 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
```

Comment:
@flat35hd99 Where is the `steps.yarn-cache-dir-path`?

Reply:
[this commit](https://github.com/remap-keys/remap/pull/401/commits/1d5c7dd8ed112da9a03b3f1288eb4b945b001142) contain this problem.